paddle/fluid/framework/inplace_op_inference_test.cc

```diff
@@ -202,7 +207,7 @@ ir::Node* GetNodeFromGraph(ir::Graph* g, std::string name) {
 
 std::unique_ptr<ir::Graph> test_SingleOpInplaceInToOut(
     std::unique_ptr<ir::Graph> g) {
-  std::unique_ptr<details::InplacePass> pass(new details::InplacePass());
+  auto pass = ir::PassRegistry::Instance().Get("inplace_pass");
```

Comment:
"auto pass = CreateInplacePass()" to de-duplicate code?

Reply:
Done.